hpeOneView/resources/servers/enclosure_groups.py

```diff
@@ -38,9 +38,7 @@ class EnclosureGroups(Resource):
     DEFAULT_VALUES = {
         '800': {"type": "EnclosureGroupV7"},
         '1000': {"type": "EnclosureGroupV7"},
-        '1200': {"type": "EnclosureGroupV8"},
-        '1600': {"type": "EnclosureGroupV8"},
-        '1800': {"type": "EnclosureGroupV8"}
```

Comment:
which api versions are having this issue..?

Reply:
its not supported from 1600 onwards